bin/test/fake_process_manager.dart

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Chromium Authors. All rights reserved.
+// Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
Are we changing all the headers to Flutter copyright, or just this file? And 2020?

Reply:
LOL, no.  I just copied this from another repo (process_runner).  Reverted copyright.